src/Raven.Server/Smuggler/Migration/Importer.cs

```diff
@@ -92,6 +92,13 @@ private async Task SaveLastState(long operationId, ImportInfo previousImportInfo
                         smugglerResult.TimeSeries = new SmugglerProgressBase.CountsWithSkippedCountAndLastEtag();
                     }
 
+                    if ((_buildVersion >= 4000 && _buildVersion <= 54133) || (_buildVersion >= 6000 && _buildVersion <= 60035))
+                    {
+                        // prevent NRE, time series deleted ranges were added in 5.4.201 and 6.0.105 
```

Comment:
@arekpalinski are these the right numbers to put here?

Reply:
```csharp
if ((_buildVersion >= 40000 && _buildVersion <= 54133) || (_buildVersion >= 60000 && _buildVersion <= 60035) || (_buildVersion >= 40 && _buildVersion < 54))
```